openslides_backend/action/mixins/create_action_with_dependencies.py

```diff
@@ -20,7 +20,7 @@ def base_update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         additional_relation_models = {
             FullQualifiedId(self.model.collection, instance["id"]): instance,
         }
-        for ActionClass in self.dependencies:
+        for index, ActionClass in enumerate(self.dependencies):
```

Comment:
This also seems like it doesn't belong in this PR. Also I think the cleaner solution would be the change the return type of the `payload_method` to an array of objects. Dealing with explicit indices is always a hassle...

Reply:
done

Reply:
Changed. The _index _ is gone and the expected return value is now a List of dict